src/main/java/com/solvd/RentalCompany/jaxb/JaxbReader.java

```diff
@@ -0,0 +1,28 @@
+package com.solvd.RentalCompany.jaxb;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+
+public class JaxbReader {
+
+    private final static Logger logger = LogManager.getLogger(JaxbReader.class);
+
+    public static void read(){
```

Comment:
if I understand correctly, then I can simply replace the return type of the method with RentalCompany and extract all objects from a variable of this type, since the RentalCompany class contains lists of all types of objects and getters for them.